### PR TITLE
feat: enable mounting external EFS volumes in Copilot jobs & services

### PR DESCRIPTION
**Pull request by bvtujo**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: templates/workloads/services/backend/cf.yml [hunk 1]
```diff
@@ -56,11 +56,11 @@ Resources:
...
             Timeout: {{.HealthCheck.Timeout}}
 {{- end}}
-{{- if .Storage}}
+{{- if .Storage -}}
 {{include "mount-points" . | indent 10}}
-{{- end}}
+{{- end -}}
 {{include "sidecars" . | indent 8}}
-{{- if .Storage}}
+{{- if .Storage -}}
```
Comment (efekarakus) [inline templates/workloads/services/backend/cf.yml:63]: oh does adding "-}}" remove an extra new line for these if blocks?

**Review by efekarakus — APPROVED** [review]

**Comment by efekarakus** [comment]
I assume we'll update the docs in a separate PR?

File: templates/workloads/services/backend/cf.yml (hunk 1 above)
Comment (bvtujo) [reply templates/workloads/services/backend/cf.yml:63]: yeah, that's right. It trims whitespace until the next directive. In these sections of template where there are multiple lines of interpolations one after another sometimes an extra newline can sneak in. The trailing `-}}` fixes that. I don't want to do it everywhere but it's helpful sometimes to clean up the rendered CF.

**Comment by bvtujo** [comment]
Yeah, I'm working on that one right now.

**Review by iamhopaul123 — APPROVED** [review]
:ship:

**Pull request merged**